include/dt-bindings/spi/spi.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2021 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_SPI_SPI_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_SPI_SPI_H_
+
+/**
+ * @brief SPI Interface
+ * @defgroup spi_interface SPI Interface
+ * @ingroup io_interfaces
+ * @{
+ */
+
+/**
+ * @brief SPI duplex mode
+ *
+ * Some controllers support half duplex transfer, which results in 3-wire usage.
+ * By default, full duplex will prevail.
+ */
+#define SPI_FULL_DUPLEX		(0 << 11)
+#define SPI_HALF_DUPLEX		(1 << 11)
```

Comment:
only first define gets documented, see related previous code

Reply:
Actually this is on all multiple defines, I'll make a dedicated commit for it